lms/static/coffee/src/instructor_dashboard/util.coffee

```diff
@@ -1,3 +1,4 @@
+# TODO - lots of safe-commit-linter failures in this file - how can I get HtmlUtils into a coffeescript file?
```

Comment:
@robrap - I may need your assistance on this one. This is a utility file that I changed ~30 lines in, that is just riddled with `javascript-concat-html` and `javascript-jquery-html` errors.

Is there a simple way to get `HtmlUtils` into this file to fix these issues?